nilearn/connectome/connectivity_matrices.py

```diff
@@ -218,19 +220,126 @@ def sym_to_vec(symmetric, discard_diagonal=False):
         The output flattened lower triangular part of symmetric. Shape is
         (..., n_features * (n_features + 1) / 2) if discard_diagonal is False and
         (..., (n_features - 1) * n_features / 2) otherwise.
+    """
+    return sym_matrix_to_vec(symmetric=symmetric, discard_diagonal=discard_diagonal)
+
+
+def sym_matrix_to_vec(symmetric, discard_diagonal=False):
+    """Return the flattened lower triangular part of an array.
+
+    If diagonal is kept, diagonal elements are divided by sqrt(2) to conserve
+    the norm.
+
+    Acts on the last two dimensions of the array if not 2-dimensional.
+
+    .. versionadded:: 0.3
+
+    Parameters
+    ----------
+    symmetric : numpy.ndarray, shape (..., n_features, n_features)
```

Comment:
well it accepts lists but also numpy arrays of shape nD
Do you think I should reformulate the doc? because actually in practice people will use this for a list of 2D matrices (or the equivalent 3D array)